CHANGES.md

```diff
@@ -18,6 +18,16 @@
 
 # Markdown Mode 2.4
 
+*Sep 02, 2020*
+
+*   Improvements:
```

Comment:
- Please add these change information to [Markdown Mode 2.5 section](https://github.com/jrblevin/markdown-mode/blob/master/CHANGES.md#markdown-mode-25)